lib/datadog/tracing/contrib/grape/endpoint.rb

```diff
@@ -268,7 +268,7 @@ def endpoint_expand_path(endpoint)
               namespace = endpoint.routes.first && endpoint.routes.first.namespace || ''
 
               parts = (namespace.split('/') + route_path).reject { |p| p.blank? || p.eql?('/') }
-              parts.join('/').prepend('/')
+              parts.join('/').delete_prefix('/').prepend('/')
```

Comment:
Hey @jethrodaniel thanks for your contribution 👍🏼 

I think given that we already have a parts with a first part to be prefixed, adding and removing `/` looks excessive. WDYT if we just check before prepending if the first char is a `/`?

Something like:

```ruby
path = (namespace.split('/') + route_path).reject { |p| p.blank? || p.eql?('/') }.join('/')
path.prepend('/') if path[0] != '/'
```

Reply:
+1 for the more efficient approach - will update when I get time. Thanks!